File: test/nullValue/nullValue_test.js
const specimen = require("../../js/dist");

let codeboxSet = specimen.makeCodeboxSet({
  nullValue: (s, input) => {
    let _ = input.nullValue
  },
});

specimen.run(codeboxSet, [
  specimen.readLocalFile("nullValue_data.yaml", __dirname),
]);

File: rust/tests/null_value_test.rs
#[derive(Debug, Default)]
struct Runner {}

impl specimen::Evaluator for Runner {
    fn evaluate(&mut self, tile: &specimen::Dict) -> bool {
        match tile.get("nullValue") {
            Some(_) => true,
            _ => false,
        }
    }
}

#[test]
fn test_null_value() {
    specimen::run(
        Box::new(Runner::default()),
        &[specimen::file::File::read_local_file(
            "../test/counter/counter_data.yaml",
        )],
    )
}
